style.css

```diff
@@ -0,0 +1,91 @@
+.burger {
+	background-color: salmon;
+	display: none;
+	height: 3px;
+	position: relative;
+	width: 10px;
+}
+
+.burger:before {
+	content: " ";
+	background-color: salmon;
+	height: 3px;
+	position: absolute;
+	top: 5px;
+	width: 10px;
+}
+
+.burger:after {
+	bottom: 5px;
+	content: " ";
+	background-color: salmon;
+	height: 3px;
+	position: absolute;
+	width: 10px;
+}
+
+@media (min-width: 320px) {
+	.nav {
+		display: none;
+	}
+}
+
+@media (min-width: 320px) {
+	.burger {
+		display: block;
+	}
+}
+
+@media (min-width: 1024px) {
+	.burger {
+		display: none;
+	}
+}
+
+@media (min-width: 1024px) {
+	.nav {
+		display: inline;
+	}
+}
+
+@media (min-width: 320px) {
+	.wrap__title {
+		font-size: 10px;
+		background-color: darkblue;
+	}
+}
+
+@media (min-width: 481px) {
+	.wrap__title {
+		font-size: 20px;
+		background-color: blue;
+	}
+}
+
+@media (min-width: 641px) {
+	.wrap__title {
+		font-size: 30px;
+		background-color: lightblue;
+	}
+}
+
+@media (min-width: 961px) {
+	.wrap__title {
+		font-size: 40px;
+		background-color: darkred;
+	}
+}
+
+@media (min-width: 1024px) {
+	.wrap__title {
+		font-size: 50px;
+		background-color: red;
+	}
+}
+
+@media (min-width: 1281px) {
+	.wrap__title {
+		font-size: 60px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5813014)